src/main/scala/mesosphere/marathon/api/v2/TasksResource.scala

```diff
@@ -92,7 +94,7 @@ class TasksResource @Inject() (
 
     val enrichedTasks: Iterable[EnrichedTask] = result(futureEnrichedTasks)
     ok(jsonObjString(
-      "tasks" -> enrichedTasks
+      "tasks" -> enrichedTasks.toIndexedSeq.toRaml
```

Comment:
why not `.toVector`?

Reply:
No specific reason. Mostly because it is more general than a vector.